packages/react/src/components/form/Radio/RadioElement.tsx

```diff
@@ -128,7 +185,11 @@ const Element = ({
   };
 
   return (
-    <RadioElement>
+    <RadioElement
+      data-variant={variant}
+      {...(isChecked ? { "data-checked": true } : {})}
```

Comment:
Just for information, if you want you can just pass `isChecked` value to the `data-checked` attribute and check from the CSS if the value is truthy.

```css
&[data-checked="true"] {...}
```

Reply:
Noice, pas besoin de passer un isChecked.toString() ? j'avais essayé un truc du style

Reply:
je t'avoue qu'au début j'avais une solution bien plus crade dans le CSS donc ça m'a pas trop dérangé de faire comme ça hahaha